.github/workflows/ci-app-examples.yml

```diff
@@ -86,10 +86,11 @@ jobs:
       - name: Install Yarn
         run: npm install -g yarn
 
-      - name: Install Lightning package
+      - name: Install package
```

Comment:
```suggestion
      - name: Install Lightning package
```

Reply:
This step also installs the lightning app package